HIP/hip-514-missing-hts-precompile-functions.md

```diff
@@ -0,0 +1,245 @@
+| `RoyaltyFee`                      | The royalty fee for the token         |
+| `address feeCollectorAccountId`   | The account to receive the custom fee |
+
+* Note that a valid `CustomFee` will have **only one** valid fee i.e. `FixedFee` or `FractionalFee` or `RoyaltyFee`.
+
+#### TokenInfo
+`TokenInfo` struct defines the basic properties of a Fungible or Non-Fungible Hedera Token:
+
+| Field                     | Meaning                                                                                           |
+|---------------------------|---------------------------------------------------------------------------------------------------|
+| `HederaToken token`       | The hedera token.                                                                                 |
+| `uint64 totalSupply`      | The number of tokens (fungible) or serials (non-fungible) of the token                            |
+| `bool deleted`            | Specifies whether the token is deleted or not                                                     |
+| `bool defaultKycStatus`   | Specifies whether the token kyc was defaulted with KycNotApplicable (true) or Revoked (false)     |
+| `bool pauseStatus`        | Specifies whether the token is currently paused or not                                            |
+| `CustomFee[] customFees`  | The custom fees collected when transferring the token                                             |
+| `string ledgerId`         | The ID of the network ledger                                                                      |
+
+Note that the tokenId and type of token (fungible or non-fungible) is not specified in the `TokenInfo` struct - it is implied 
+by the specific precompile function being called (see the functions below).
+
+#### FungibleTokenInfo
+`FungibleTokenInfo` struct defines the basic properties of a Fungible Hedera Token:
+
+| Field                 | Meaning                                               |
+|-----------------------|-------------------------------------------------------|
+| `TokenInfo tokenInfo` | The shared hedera token info                          |
+| `uint32 decimals`     | The number of decimal places a token is divisible by  |
+
+#### NonFungibleTokenInfo
+`NonFungibleTokenInfo` struct defines the basic properties of a Non Fungible Hedera Token:
+
+| Field                     | Meaning                                                                                       |
+|---------------------------|-----------------------------------------------------------------------------------------------|
+| `TokenInfo tokenInfo`     | The shared hedera token info                                                                  |
+| `uint64 serialNumber`     | The serial number of the nft                                                                  |
+| `address accountId`       | The account id specifying the owner of the non fungible token                                 |
```

Comment:
Would it be clearer for `accountId` -> `ownerId`?

Reply:
Can do

Reply:
Updated